tests/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,494 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(OPENSSL_DIR ${PROJECT_SOURCE_DIR}/3rdparty/openssl/openssl)
+
+add_subdirectory(host)
+if (BUILD_ENCLAVES)
+  add_subdirectory(enc)
+endif ()
+
+macro (copy_test_data OLD_FILE NEW_FILE)
+  if (UNIX)
+    add_custom_command(OUTPUT ${NEW_FILE} COMMAND ${CMAKE_COMMAND} -E copy
+                                                  ${OLD_FILE} ${NEW_FILE})
+  else ()
+    # On Windows, invoking the dos2unix utility from git to replace
+    # CRLF by LF in the test file.
+    add_custom_command(OUTPUT ${NEW_FILE} COMMAND ${OE_DOS2UNIX} -n ${OLD_FILE}
+                                                  ${NEW_FILE})
+  endif ()
+
+  # Add the copied files to the list so that the custom target
+  # openssl_test_data can take dependency on.
+  list(APPEND test_data_list ${NEW_FILE})
+endmacro (copy_test_data)
+
+# Skip on Windows if dos2unix is not supported.
+if (UNIX OR OE_DOS2UNIX)
```

Comment:
Only few tests require those files with LR. So the current logic is adding check the preset of dos2unix to those tests only. If not present, we will skip these tests but still include those unaffected tests.

Reply:
As I remarked earlier, the OE_DOS2UNIX check can be removed here by making dos2unix a hard requirement satisfied via the git installation.
Doing so means that the set of tests run on Windows will be the same irrespective of which machine it is run on, instead of silently disabling some tests.